Source/WebCore/animation/ScrollTimeline.h

```diff
@@ -54,7 +53,8 @@ class ScrollTimeline : public AnimationTimeline {
     static Ref<ScrollTimeline> create(Scroller, ScrollAxis);
 
     virtual Element* source() const;
-    void setSource(const Element*);
+    void setSource(Element*);
+    void setSource(const WeakStyleable&);
```

Comment:
I don't think this is a good idea. We should only use a `Styleable` just like we do throughout the animations code.

Reply:
Made this use Styleable.